README.md

```diff
@@ -61,4 +61,9 @@ Where is the best pizza in a given city? This is a community project. Pull reque
 * [Bitondo's Pizzeria](https://plus.google.com/111316627016040776065/about?gl=ca&hl=en)
 
 ## Buffalo, NY
+
 * [Pizza By Molino's](http://www.pizzabymolinos.com/)
+
+## California
+
+* [California Pizza Kitchen (CPK)](http://www.cpk.com/)
```

Comment:
[This place](http://www.pizzanapoletana.org/showassoc_eng.php?id=472) is the best pizza in San Jose.